src/main/resources/reference.conf

```diff
@@ -31,32 +31,36 @@ jicofo {
 
     // RTP header extensions
     rtp-extensions {
+      audio-level {
```

Comment:
You've got the extension a different order in reference.conf than you do in CodecConfig.kt and CodecConfigTest.kt - is there a reason?

Reply:
I ordered these by ID, the order in the code was arbitrary

Reply:
It's just that having them in a different order in the two places makes it a bit difficult to visually confirm that everything's covered. Not a big deal though.